### DISCUSSION TITLE
Upcoming release [Apr 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5560] [Discord Threads]
> 🙋‍♂️ **replavar**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### https://github.com/nocodb/nocodb/milestone/27?closed=1   
####    
#### - better filters for date time   
#### - short cut keys for navigation with in expanded record   
#### - bug fixes   
#### - undo (wip)   
####    
#### hopefully by mid of next week  
#  
> 🙋‍♂️ **replavar**  
#### Thanks for answering!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1087746563012575328)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
